man/submit_ipr.Rd

```diff
@@ -4,7 +4,7 @@
 \alias{submit_ipr}
 \title{Submit IPRscan analysis for protein sequences (multifasta input)}
 \usage{
-submit_ipr(path2seq, outfolder, email)
```

Comment:
any reason to go back to user_email?

Reply:
Nope, you're right. Plain email is better. 

Just happened because I started the rework before we made that change. 

I'll update this